saleor/webhook/transport/synchronous/circuit_breaker/const.py

```diff
@@ -0,0 +1,6 @@
+class BreakerMode:
+    NONE = "none"
+    FIXED = "fixed"
```

Comment:
I'd personally leave `FIXED` mode out for now. Currently I'm not seeing any application for this mode, when we need it we can always add it.